### PR TITLE
Add Identity and EF

### PR DESCRIPTION
**Pull request by suhasj**
@davidfowl @smitpatel @muratg

### CODE REVIEW

File: src/JabbR/Models/JabbrContext.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        {
+            options.UseSqlServer();
+        }
+
+        protected override void OnModelCreating(ModelBuilder builder)
+        {
+            base.OnModelCreating(builder);
+        }
+    }
+}
```
Comment (muratg) [inline src/JabbR/Models/JabbrContext.cs]: Nit: New line

**Comment by muratg** [comment]
:shipit:

File: src/JabbR/Models/JabbrContext.cs (hunk 1 above)
Comment (suhasj) [inline src/JabbR/Models/JabbrContext.cs]: @muratg should there by a new line at the end of the file ?

Comment (muratg) [inline src/JabbR/Models/JabbrContext.cs]: Not super important but better to be consistent with the other files :)

File: src/JabbR/config.json [hunk 2]
```diff
@@ -1,2 +1,12 @@
 ﻿{
+    "Data": {
+        "DefaultConnection": { 
+            "ConnectionString": "Server=(localdb)\\mssqllocaldb;Database=aspnet5-Jabbrv2-1;Trusted_Connection=True;MultipleActiveResultSets=true"
```
Comment (davidfowl) [inline src/JabbR/config.json]: Why is the database name so bad?

File: src/JabbR/Models/JabbrContext.cs (hunk 1 above)
Comment (suhasj) [inline src/JabbR/Models/JabbrContext.cs]: sure I'll check that and correct it

File: src/JabbR/config.json (hunk 2 above)
Comment (suhasj) [inline src/JabbR/config.json]: I can make it `JabbrDb` ? I modeled on the lines of what templates have

File: src/JabbR/Models/JabbrContext.cs [hunk 3]
```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.AspNet.Identity.EntityFramework;
+using Microsoft.Data.Entity;
+using Microsoft.Data.Entity.Metadata;
+
+namespace JabbR.Models
+{
+    public class JabbrContext :IdentityDbContext<ChatUser>
+    {
+        private static bool _created = false;
```
Comment (smitpatel) [inline src/JabbR/Models/JabbrContext.cs:9]: Where is _created being used?

Comment (suhasj) [inline src/JabbR/Models/JabbrContext.cs:9]: good catch. Fixed

**Comment by suhasj** [comment]
Updated with review feedback

**Comment by suhasj** [comment]
pushed

**Pull request closed without merging**